### PR TITLE
Initial add of media streaming

### PR DESCRIPTION
**Pull request by moirf**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: CallAutomation_MediaStreaming/WebSocketListener/Program.cs [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                    catch (Exception ex)
+                    {
+                        httpListenerContext.Response.StatusCode = 500;
+                        httpListenerContext.Response.Close();
+                        Console.WriteLine($"Exception -> {ex}");
+                        return;
+                    }
+
+                    WebSocket webSocket = websocketContext.WebSocket;
+                    Dictionary<string, FileStream> audioDataFiles = new Dictionary<string, FileStream>();
```
Comment (moirf) [inline CallAutomation_MediaStreaming/WebSocketListener/Program.cs:44]: If this is not going to create object each time its get into this loop?

File: CallAutomation_HeroSample/.gitignore [hunk 2]
```diff
@@ -0,0 +1,354 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
```
Comment (Ninika) [inline CallAutomation_HeroSample/.gitignore:3]: This file should not be checked in

Comment (Sulochanan) [reply CallAutomation_HeroSample/.gitignore]: Resolved

File: CallAutomation_MediaStreaming/Readme.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+1. Open an instance of PowerShell, Windows Terminal, Command Prompt or equivalent and navigate to the directory that you'd like to clone the sample to.
+2. git clone https://github.com/Azure-Samples/Communication-Services-dotnet-quickstarts.git.
+
+### Locally running the media streaming WebSocket app
+
+1. Go to CallAutomation_MediaStreaming folder and open `IncomingCallMediaStreaming.sln` solution in Visual Studio.
+2. Select and run the `WebSocketListener` project, an application for listening media stream on websocket.
+3. Used the dev tunnel to get public URL for `WebSocketListener` app, as this websocket URL needed for `MediaStreamingTransportURI` configuration.
+
+### Publish  the Incoming call media streaming to Azure WebApp
```
Comment (antonsamson-msft) [inline CallAutomation_MediaStreaming/Readme.md:55]: Is it possible for Contoso to run both the WebSocket listener and call handlers locally?

**Pull request merged**